packages/compass-crud/src/components/table-view/types-dropdown.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import TypeChecker from 'hadron-type-checker';
+import { Select, Option, css, spacing } from '@mongodb-js/compass-components';
+
+const CASTABLE_TYPES = TypeChecker.castableTypes(true);
+
+const selectStyles = css({ minWidth: spacing[3] * 10 });
+
+const TypesDropdown = ({ element }) => {
```

Comment:
Is there a reason we don't want this file in typescript? `compass-crud` is on the new config. We might be able to use some of the types from `hadron-document` as well.  Same q with the `types-dropdown.spec.jsx`